src/Fixer/Casing/LowercaseConstantsFixer.php

```diff
@@ -24,8 +27,20 @@
  *
  * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
  */
-final class LowercaseConstantsFixer extends AbstractFixer
+final class LowercaseConstantsFixer extends AbstractFixer implements ConfigurationDefinitionFixerInterface
```

Comment:
I find it a bit strange to have a `Lowercase...Fixer` that _uppercases_ things, but I guess that's how it has to be seeing that renaming is not an option.

Reply:
Indeed, had the same thing in mind. I will update accordingly today evening.

Reply:
The reason `lowercase_constants: false` doesn't work is because that disables the rule (meaning no fixing either way). But that was not my point, as even if that were a way to configure it, it would still be a `Lowercase...Fixer` that uppercases things.

And like I said, renaming is not an option (because of BC), so there is no solution.

Reply:
Mmh... I understand but what I had in mind was the following:

* Not having `lowercase_constants`: Do not do anything
* Having `lowercase_constants: false` means to uppercase them
* Having `lowercase_constants: true` means to lowercase them

Reply:
Ok, thanks for the explanation, make sense.